services/web-server/src/graphql/Hooks.graphql

```diff
@@ -72,11 +72,17 @@ type DeleteHook {
   hookId: ID
 }
 
+input HookBinding {
```

Comment:
Does this work for you? I think it should be `type`, not `input`.

Reply:
On adding type web-server was throwing error and changing it to type was suggested. So I changed it to type

Reply:
You are correct. My mistake :)